pkg/exchange/okex/query_closed_orders_test.go

```diff
@@ -0,0 +1,38 @@
+package okex
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/c9s/bbgo/pkg/testutil"
+	"github.com/c9s/bbgo/pkg/types"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_QueryClosedOrders(t *testing.T) {
+
+	key, secret, passphrase, ok := testutil.IntegrationTestWithPassphraseConfigured(t, "OKEX")
+	if !ok {
+		t.Skip("Please configure all credentials about OKEX")
+	}
+
+	e := New(key, secret, passphrase)
+
+	queryOrder := types.OrderQuery{
+		Symbol: "BTC-USDT",
```

Comment:
types.OrderQuery is a global type, so the symbol value will be "BTCUSDT" not "BTC-USDT"

Reply:
Yes, you're right, I use `toLocalSymbol` to fix it, and I'm afraid my previous PR made the same error, Will fix it in coming PR, already fix QueryTrades(), QueryClosedOrders(), QueryOrder()